changes_test.go

```diff
@@ -2494,6 +2494,59 @@ func (s *changesSuite) TestChangeDescriptions(c *gc.C) {
 				"add unit ror/1 to 2/kvm/0",
 				"add unit ror/2 to new machine 3",
 			},
+		}, {
+			description: "machine natural sorting",
+			bundleContent: `
+                applications:
+                    ubu:
+                        charm: cs:ubuntu
+                        num_units: 13
+                        to: [0,1,2,3,4,5,6,7,8,9,10,11,12]
+                machines:
+                    0:
+                    1:
+                    2:
+                    3:
+                    4:
```

Comment:
Did this test really need all 0-12 machines? Would just having just 2 and 10 still show the problem?

Reply:
Perhaps not entirely, but matches bug report :)